addons/spectator/functions/fnc_ui_handleLoad.sqf

```diff
@@ -0,0 +1,25 @@
+/*
+ * Author: SilentSpike, Jonpas
+ * Function used to handle load event.
+ *
+ * Arguments:
+ * 0: Spectator display <DISPLAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * _this call ace_spectator_fnc_ui_handleLoad
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_display"];
+
+uiNamespace setVariable [QGVAR(display), _display];
+
+// Handle ACRE2 Toggle Spectator (if present)
+if (["acre_sys_radio"] call EFUNC(common,isModLoaded)) then {
```

Comment:
`!isNil "acre_api_fnc_addDisplayPassthroughKeys"` and merge now, not having to wait for ACRE update + bwc?

Reply:
Have to add it here and `isNil` where it was handled manually before.
Previous manual handling will continue to work after ACRE2 update, without any change from how it currently works (only supports first keybind).

Reply:
Do we want that backwards compatibility? I don't really see a reason to be honest, people should switch to updated version of ACRE2.